packages/prci/src/StatusViewItem.ts

```diff
@@ -292,36 +293,74 @@ export class StatusViewItem extends LitElement {
   }
 
   transparencyProvTemplate(demand: PrivacyResponseItem) {
+    console.log(demand);
     const answer = demand.answer as {
       'AFFILIATION.selector_1': Provenance[];
       'AFFILIATION.MEMBERSHIP.selector_2': Provenance[];
       'FINANCIAL.BANK-ACCOUNT.selector_3': Provenance[];
     };
+    console.log(Object.keys(demand.answer as Object));
+    console.log(Object.entries(demand.answer as Object));
```

Comment:
Would be better to avoid console.log in the future. Ok for this patch/demo release, but please do a global review afterwards to remove them (or even, add a "debug" option, but this would require injecting a custom logger, which may be too much for now).

Reply:
Hmm, until now eslint would stop a commit if there were any issues, weird. But yes I will remove them